packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -162,13 +162,15 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
 
     private toInstrumentedPushTokenFetcher(
         logger: ITelemetryLogger,
+        resolvedUrl: IOdspResolvedUrl,
         tokenFetcher: PushTokenFetcher,
     ): (options: TokenFetchOptions) => Promise<string | null> {
         return async (options: TokenFetchOptions) => {
             return PerformanceEvent.timedExecAsync(
                 logger,
                 { eventName: "GetWebsocketToken" },
-                async (event) => tokenFetcher(options.refresh, options.claims).then((tokenResponse) => {
+                async (event) => tokenFetcher(resolvedUrl.siteUrl, options.refresh, options.claims)
```

Comment:
Maybe time to create an interface and pass as single object? :)

Reply:
Agreed. I went ahead and made a few more changes to streamline type definitions that hopefully make things easier to understand and consume.